api/api/spec/spec.yaml

```diff
@@ -5279,7 +5279,16 @@ components:
       schema:
         type: string
         format: xml_filename_path
-
+    resource_type:
+      in: query
+      name: resource_type
+      description: "Filter by process resource type"
```

Comment:
Find another description. Keep in mind that a "security resource" could be a role, a user, a policy... This description is not accurate enough.

Reply:
Updated